tests/test_venue_request.py

```diff
@@ -315,6 +319,55 @@ def test_venue_recruitment(self, client, test_client, selenium, request_page, ve
         assert messages and len(messages) == 1
         assert messages[0]['content']['subject'] == '[TestVenue@OR2030] Invitation to serve as reviewer'
         assert messages[0]['content']['text'].startswith('Dear Reviewer Two,\n\nYou have been nominated by the program chair committee of Theoretical Foundations of RL Workshop @ ICML 2020 to serve as reviewer.')
+    
+    def test_venue_recruitment_tilde_IDs(self, client, test_client, selenium, request_page, venue, helpers):
+
+        # Test Reviewer Recruitment
+        request_page(selenium, 'http://localhost:3030/forum?id={}'.format(venue['request_form_note'].id), test_client.token)
+        recruitment_div = selenium.find_element_by_id('note_{}'.format(venue['request_form_note'].id))
+        assert recruitment_div
+        reply_row = recruitment_div.find_element_by_class_name('reply_row')
+        assert reply_row
+        buttons = reply_row.find_elements_by_class_name('btn-xs')
+        assert [btn for btn in buttons if btn.text == 'Recruitment']
+        helpers.create_user('reviewer_one_tilde@mail.com', 'Reviewer', 'OneTilde') #Added by me 
+        helpers.create_user('reviewer_two_tilde@mail.com', 'Reviewer', 'TwoTilde') #Added by me 
+        reviewer_details = '''~Reviewer_OneTilde1\n~Reviewer_TwoTilde1'''
+        
+        recruitment_note = test_client.post_note(openreview.Note(
+            content={
+                'title': 'Recruitment',
+                'invitee_role': 'reviewer',
+                'invitee_details': reviewer_details, 
+                'invitation_email_subject': '[' + venue['request_form_note'].content['Abbreviated Venue Name'] + '] Invitation to serve as {invitee_role}',
+                'invitation_email_content': 'Dear {name},\n\nYou have been nominated by the program chair committee of Theoretical Foundations of RL Workshop @ ICML 2020 to serve as {invitee_role}.\n\nACCEPT LINK:\n\n{accept_url}\n\nDECLINE LINK:\n\n{decline_url}\n\nCheers!\n\nProgram Chairs'
+            }, # Will this still work to get the name like {name} or do we need to do some special fetching thing
```

Comment:
this still works, because when we pass a tilde ID instead of a email,name pair, we get the name by replacing '_' with ' ' and remove the tilde: https://github.com/openreview/openreview-py/blob/00eac11bdd76cdb47ddfb5246537d32799dc2ba8/openreview/conference/builder.py#L1359